acceptance_tests/features/fixtures.py

```diff
@@ -165,6 +167,7 @@ def _setup_data_with_internal_user_and_collection_exercise_to_specific_status(co
     context.dates = response['dates']
 
     create_internal_user(context)
+    create_alternate_internal_user(context)
```

Comment:
Good call, changing to a parameterised call which can then support more than 2 internalusers

Reply:
It does need a separate function. What it does is to create a user name using random data and assign that to a variable held in the context so that that user is available for later tests.So the setup creates context.internal_user and context.alternate_internal_user and those ids are available for any step that wants to use them . Up to now the only use case for the second internal user is secure messaging.

Reply:
I agree with @insacuri 's comment, but I don't see it as a mandatory change: just something that could be improved. I don't want to add to tech debt, but the acceptance tests need some major re-working anyway. I'll leave it to @AndrewTorrance 's judgement if and when he improves this.